plugins/feeds/public/otx_alienvault.py

```diff
@@ -88,22 +88,40 @@ def analyze(self, item):
 
                 obs.tag(tags)
                 obs.add_context(self.name, context)
-
+                investigation.link_to(obs, "Observed", "OTXAlienVault")
             elif type_ind in entity.EntityType:
-                entity.Entity(
+                ent = entity.Entity(
                     name=otx_indic["indicator"],
                     type=self._TYPE_MAPPING.get(otx_indic["type"]),
                 ).save()
-
+                investigation.link_to(ent, "Observed", "OTXAlienVault")
             elif type_ind in indicator.IndicatorType:
                 if type_ind == indicator.IndicatorType.yara:
-                    indicator.Indicator(
+                    ind_obj = indicator.Indicator(
                         name=f"YARA_{otx_indic['indicator']}",
-                        pattern=otx_indic["content"],
+                        pattern="OTX",
                         type=indicator.IndicatorType.yara,
                         location="OTX",
                         diamond=indicator.DiamondModel.capability,
-                    ).save()
+                    )
```

Comment:
Let's move this to after we've parsed the Yara rule, and use that info directly in the constructor of `Indicator`